src/main/groovy/org/ajoberstar/gradle/git/publish/GitPublishPlugin.groovy

```diff
@@ -20,7 +20,6 @@ import java.nio.file.Files
 import groovy.transform.PackageScope
 
 import org.ajoberstar.grgit.Grgit
-import org.ajoberstar.grgit.exception.GrgitException
 import org.ajoberstar.grgit.operation.FetchOp
 import org.ajoberstar.grgit.operation.ResetOp
 import org.eclipse.jgit.errors.RepositoryNotFoundException
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The [org.eclipse.jgit.errors.RepositoryNotFoundException] import is never referenced [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=grvy%3Aorg.codenarc.rule.imports.UnusedImportRule)